src/SFA.DAS.ProviderApprenticeshipsService.Application/Commands/SubmitCommitment/SubmitCommitmentCommand.cs

```diff
@@ -10,6 +12,8 @@ public class SubmitCommitmentCommand : IAsyncRequest
 
         public string Message { get; set; }
 
-        public string SaveOrSend { get; set; }
+        public AgreementStatus AgreementStatus { get; set; }
```

Comment:
@leewadhams, @froberg, can we discuss this morning about where to create the task